.github/workflows/validation.yml

```diff
@@ -17,10 +17,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: codespell-project/actions-codespell@master
+      - uses: actions/setup-python@v4
```

Comment:
So that we don't have to maintain these options in multiple places, why not make a [.codespellrc](https://github.com/effigies/bids-specification/blob/0c218f714fd65c5e266d50ba51bc087c16775536/.codespellrc) and keep this as the codespell action?

Reply:
if the GH action supports that config file, that's be a superior solution, but I couldn't find docs on it: https://github.com/codespell-project/actions-codespell ... but now looking at it, this [comment](https://github.com/codespell-project/actions-codespell/issues/42#issuecomment-1013101886) suggests that it's possible.